src/MicrosoftTeams.schema.json

```diff
@@ -43,79 +43,136 @@
+            "maxLength": 16
+        },
+        "description": {
+            "type": "object",
+            "properties": {
+                "short": {
+                    "type": "string",
+                    "description": "A short description of the app used when space is limited. Maximum length is 80 characters.",
+                    "maxLength": 80
+                },
+                "full": {
+                    "type": "string",
+                    "description": "The full description of the app. Maximum length is 256 characters.",
+                    "maxLength": 256
+                }
+            },
+            "required": [
+                "short",
+                "full"
+            ]
+        },
+        "icons": {
+            "type": "object",
+            "description": "Each icon image file must be a transparent PNG, with a white or light-colored background.",
+            "properties": {
+                "44": {
+                    "type": "string",
+                    "description": "An icon for the app sized to 44x44."
+                },
+                "88": {
+                    "type": "string",
+                    "description": "An icon for the app sized to 88x88."
+                }
+            },
+            "required": [
+                "44",
+                "88"
+            ]
+        },
+        "accentColor": {
+            "type": "string",
+            "description": "A color to use in conjunction with the app's icons. The value must be a valid HTML color code starting with '#', for example `#4464ee`."
+        },
+        "galleryTabs": {
             "type": "array",
             "items": {
                 "type": "object",
                 "properties": {
-                    "id": {
+                    "configUrl": {
                         "type": "string",
-                        "description": "A unique identifier for this extension. The id must use reverse domain name notation.",
-                        "maxLength": 64
+                        "description": "The url to use when configuring the tab."
                     },
-                    "name": {
-                        "type": "string",
-                        "description": "The display name of the extension.",
-                        "maxLength": 16
+                    "canUpdateConfig": {
+                        "type": "boolean",
+                        "description": "A value indicating whether an instance of the tab's config can be updated by the user after creation.",
+                        "default": true
                     },
-                    "description": {
-                        "type": "object",
-                        "properties": {
-                            "short": {
+                    "scope": {
+                        "type": "array",
+                        "items": {
+                            "teams": {
                                 "type": "string",
```

Comment:
I don't think this is how an array of pre-defined strings is specified in a schema.

I think items should be like:
"items": {
  "enum": [ "teams", "people", "personal" ]
}